kythe/typescript/README.md

```diff
@@ -45,11 +309,11 @@ The Google TypeScript build relies heavily on TypeScript's `--declaration` flag
 to enable separate compilation. The way this works is that after compiling
 library A, we generate -- using that flag -- the "API shape" of A into `a.d.ts`.
 Then when compiling a library B that uses A, we compile `b.ts` and `a.d.ts`
-together.  The Kythe process sees the same files as well.
+together. The Kythe process sees the same files as well.
 
 What this means for indexing design is that a TypeScript compilation may see
-only the generated shape of a module, and not its internals.  For example,
-given a file like
+only the generated shape of a module, and not its internals. For example, given
```

Comment:
These whitespace changes seem fine to me but maybe put them in a different change to keep the history clear.